resources/fixtures/config/asset.yml

```diff
@@ -1 +1,2 @@
-auto-generate: false
\ No newline at end of file
+auto-generate: false
+local-minify: false
```

Comment:
Would it not be better to have this as a `minify` config which you set differently based on environment. Otherwise you are setting local environment specific setting in the global environment.

Then again we want different default behaviour for local and standard.

Perhaps this config should take an environment/list of environments for which to minify?